tests/de.rs

```diff
@@ -45,6 +45,20 @@ fn deserialize_unit_struct_u8_redis_int() {
     assert_eq!(expected, actual);
 }
 
+#[test]
+fn deserialize_bool() {
+    let v = vec![
+        Value::Data(b"0".to_vec()), Value::Data(b"false".to_vec()), Value::Data(b"False".to_vec()),
+        Value::Data(b"1".to_vec()), Value::Data(b"true".to_vec()), Value::Data(b"True".to_vec())
+    ];
+
+    let de = Deserializer::new(Value::Bulk(v));
+    let actual: Vec<bool> = Deserialize::deserialize(de).unwrap();
+
+    let expected = vec![false, false, false, true, true, true];
```

Comment:
Not that it really matters for a test, but you can avoid this allocation by storing into a tuple of bools.